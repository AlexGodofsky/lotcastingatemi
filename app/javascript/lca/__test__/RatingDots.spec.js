import React from 'react'
import RatingDots from '../utils/ratingDots.jsx'
import renderer from 'react-test-renderer'

test('RatingDots renders at all', () => {
  const component = renderer.create(
    <RatingDots rating={3} />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
