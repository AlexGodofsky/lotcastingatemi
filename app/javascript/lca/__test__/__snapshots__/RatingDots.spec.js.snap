// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingDots renders at all 1`] = `
<div
  className="sc-bdVaJa ikDpDj"
>
  <div
    className="sc-bxivhb gwefrW sc-bwzfXH jjwuFK"
  >
    ●
  </div>
  <div
    className="sc-bxivhb gwefrW sc-bwzfXH jjwuFK"
  >
    ●
  </div>
  <div
    className="sc-bxivhb gwefrW sc-bwzfXH jjwuFK"
  >
    ●
  </div>
  <div
    className="sc-htpNat KtFsv sc-bwzfXH jjwuFK"
  >
    ○
  </div>
  <div
    className="sc-htpNat KtFsv sc-bwzfXH jjwuFK"
  >
    ○
  </div>
</div>
`;
